src/profile/voyages/tag/vsca.js

```diff
@@ -39,14 +39,21 @@ try{
       if (u.ev[a] || u.ev.all !== undefined) {
 
         var c, d, e, f;
-
+	//START Customised VSCA tag call
+	var vscabaseurl ="";
+	if(utag.cfg.path =="//tags.tiqcdn.com/utag/expedia/voyages/prod/"){
+	  vscabaseurl="https://secure.analytics.voyages-sncf.com/prod/2.1/agency-funnel/vsca.js";
```

Comment:
Of course we could meet, let me know when you will be available ?
Otherwise, The purpose of this script is simple. we should populate a vsca_pageTag object on widows and then load a script https://secure.analytics.voyages-sncf.com/prod/2.1/agency-funnel/vsca.js:
Depend on the page the vsca_pageTag Object should contain some data from utag_data: 
Here is the generic format : 
vsca_pageTag = {
"config": {
        "pageId":"",
        "siteId":"agency-funnel",
        "vsca_version":"2.1",
        "country":"FR",
        "language":"fr"
},
        "contextData": {
                "pageContext":"basket",
                "packageType":"FC|FH|FHC",                                             
                "purchase": {
                        "orderPrice":120.00,
                        "purchaseId":"0102020220",
                },
                "customer": {
                        "email":"email@example.com",
                },
                "products":[{}]
        }
    };
var products = [
    {
     "productType":"Flight",
            "price":120.00, // "prix du l'epack"
            "outwardDepartureDate": "2016-04-27",
            "inwardDepartureDate": "2016-04-28",
            "travelType" :"OW | RT",
            "origin": {
                "stationCode": "ORY",
                "stationName" : "ORLY",
                "cityCode" : "PAR",
                "cityName": "PARIS"
            },
            "destination": {
                "stationCode": "LYS",
                "stationName" : "SAINT EXUPERY",
                "cityCode" : "LYN",
                "cityName": "LYON"  
           }
    },
    {
            "productType":"Car",
            "price":120.00,   // "prix du l'epack"        
            "outwardDepartureDate": "2016-04-27",
            "inwardDepartureDate": "2016-04-28",
            "agency":"Europcar",
            "carComfort":"E",
            "origin": {
                "stationCode": "PNO",
                "stationName" : "PARIS NORD",
                "cityCode" : "PAR",
                "townLabel": "PARIS"
            },
            "destination": {
                "stationCode": "PNO",
                "stationName" : "PARIS NORD",
                "cityCode" : "PAR",
                "cityName": "PARIS"    
           }
    },
    {
            "productType":"Hotel",
            "price":120.00,            
            "hotelId":"2431617",
            "hotelName":"Renaissance Paris Arc de Triomphe Hotel",
            "starRating":"5.3",
            "checkInDate": "2016-04-27",
            "checkOutDate": "2016-04-28",
            "cityName": "PARIS",
            "rooms":[ {
                "adults": 1,
                "children": 0
            }],       
    }
];